tests/evaluate/test_evaluation.py

```diff
@@ -568,6 +568,26 @@ def test_pandas_df_regressor_evaluation(linear_regressor_model_uri):
         assert v == saved_metrics[k]
 
 
+def test_spark_df_regressor_evaluation(linear_regressor_model_uri):
```

Comment:
can we also write a test for pandas df?

Reply:
it's existing,  I'm adding the datasets check to that case.

Reply:
I am stacking these tests into one parameterized test since they are doing the same thing.